tests/test_suite-vm.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import argparse
+
+from ssg_test_suite import test_suite
+import ssg_test_suite.test_env
+
```

Comment:
I completely dislike that this is being splitted out. Please make only single script `test-suite.py`.

Reply:
Done according to our discussion.